Speckle_Engine/UtilityMethods.cs

```diff
@@ -0,0 +1,70 @@
+﻿using BH.oM.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BH.Engine.Speckle
+{
+    public static class UtilityMethods
+    {
+        public static IEnumerable<IBHoMObject> FilterIBHoMObjects(IEnumerable<object> objects)
```

Comment:
Trying to avoid Utility classes. Is there any obstacle in Moving this to the `Engine.Speckle.Modify` class?

Reply:
Agreed.

Reply:
Query instead